api/src/main/java/ca/bc/gov/educ/api/gradstudent/model/transformer/ReportGradStudentTransformer.java

```diff
@@ -46,6 +47,9 @@ public List<ReportGradStudentData> transformToDTO (Iterable<ReportGradStudentDat
 		List<ReportGradStudentData> result = new ArrayList<>();
         for (ReportGradStudentDataEntity entity : studentStatusEntities) {
             ReportGradStudentData data = modelMapper.map(entity, ReportGradStudentData.class);
+            if(StringUtils.isNotBlank(entity.getDistrictId())) {
```

Comment:
The only thing I would suggest is making this a try catch so you don't have to check for isNotBlank as it will catch nulls and also malformed UUID strings